### PR TITLE
Add Magento 1 Image

### PR DESCRIPTION
**Pull request by kierenevans**
A magento1 image, which will run setup, etc.

### CODE REVIEW

File: magento1-nginx/fpm-5.6/usr/local/share/env/30-framework [hunk 1]
```diff
@@ -46,3 +46,10 @@ export MAGENTO_REDIS_CACHE_DATABASE=${MAGENTO_REDIS_CACHE_DATABASE:-0}
 export MAGENTO_REDIS_FULL_PAGE_CACHE_DATABASE=${MAGENTO_REDIS_FULL_PAGE_CACHE_DATABASE:-1}
 
 export MAGENTO_ADMIN_FRONTNAME=${MAGENTO_ADMIN_FRONTNAME:-admin}
+
+export MAGENTO_CREATE_ADMIN_USER=${MAGENTO_CREATE_ADMIN_USER:-$DEVELOPMENT_MODE}
+export MAGENTO_ADMIN_USERNAME=${MAGENTO_ADMIN_USERNAME:-admin}
+export MAGENTO_ADMIN_EMAIL=${MAGENTO_ADMIN_EMAIL:-admin@example.com}
```
Comment (andytson-inviqa) [inline magento1-nginx/fpm-5.6/usr/local/share/env/30-framework]: can we use admin@localhost

Comment (kierenevans) [inline magento1-nginx/fpm-5.6/usr/local/share/env/30-framework]: Sorted

**Comment by kierenevans** [comment]
Confirmed as working 🎈

**Comment by sroze** [comment]
Nice!! 👍

**Pull request merged**